app/src/operations/mail/index.ts

```diff
@@ -70,17 +66,62 @@ export default defineOperationApp({
 								text: '$t:interfaces.input-rich-text-html.wysiwyg',
 								value: 'wysiwyg',
 							},
+							{
+								text: 'Template', // TODO: i18n
+								value: 'template',
+							},
 						],
 					},
 				},
 			},
+			{
+				field: 'template',
```

Comment:
perhaps we can add `base` as the default value to be more "transparent" (in case user are unsure what would leaving it blank do), as api side does include the `template || 'base'` logic.